services/ingestion/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: "2.3"
+services:
+    ingestion:
+        build: .
+        ipc: host
+        volumes:
+            - ${INPUT_DIR}:/input/
+        command: "python3 src/pdf_ingestion.py /input 30"
```

Comment:
src/pdf_ingestion doesn't exist. Working directory is /app/ based on the Dockerfile, so just `python3 pdf_ingestion.py /input 30`